### ISSUE TITLE
kyt?

### ISSUE THREAD

**Issue by cameronroe** [r-park/todo-redux-saga#5]
Would you consider replacing the build scripts with kyt or is there any specific need that is unmet? https://github.com/NYTimes/kyt

**Comment by r-park** [comment]
I've heard of it but haven't had a chance to try it yet. Maybe this weekend I'll take a look.

**Comment by cameronroe** [comment]
Cool, yeah I think it could be super helpful to remove a lot of the setup config. Let me know how it goes!

**Comment by cameronroe** [comment]
Any luck on adding this?

**Comment by r-park** [comment]
No, sorry @cameronroe — work's been really hectic, haven't had a chance to look into it.

**Comment by cameronroe** [comment]
@r-park No worries. If I get a chance, I could do a PR. Seems like it could be useful!
